forge-gui/res/cardsfolder/upcoming/turn_inside_out.txt

```diff
@@ -0,0 +1,7 @@
+Name:Turn Inside Out
+ManaCost:R
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ +3 | SubAbility$ DBDelayedTrigger | SpellDescription$ Target creature gets +3/+0 until end of turn. When it dies this turn, manifest dread. (Look at the top two cards of your library. Put one onto the battlefield face down as a 2/2 creature and the other into your graveyard. Turn it face up any time for its mana cost if it's a creature card.)

```

Comment:
remove duplicated description part

Reply:
I meant this as description for the spell, and the next part for the delayed trigger.

Reply:
Because you used ` SpellDescription` below, you could remove that duplicated part from this line.